adapters/src/main/java/io/realm/RealmRecyclerViewAdapter.java

```diff
@@ -33,20 +33,27 @@
  * The RealmAdapter will stop receiving updates if the Realm instance providing the {@link OrderedRealmCollection} is
  * closed.
  *
- * @param <T> type of {@link RealmModel} stored in the adapter.
+ * @param <T>  type of {@link RealmModel} stored in the adapter.
```

Comment:
@dalinaum The additional space is to align it with the below line, the IDE created it when I edited something nearby, isn't that fine?